integrations/haystack/README.md

```diff
@@ -0,0 +1,305 @@
+# Haystack: Information Retrieval #
+The relevant features added as a part of the Haystack information retrieval integration are a [Haystack pipeline](/src/deepsparse/transformers/haystack/pipeline.py), an [embedding extraction pipeline](/src/deepsparse/transformers/pipelines/embedding_extraction.py), and two classes, [DeepSparseEmbeddingRetriever](/src/deepsparse/transformers/haystack/nodes.py) and [DeepSparseDensePassageRetriever](/src/deepsparse/transformers/haystack/nodes.py).
+
+These features allow a user to perform information retrieval tasks using the Haystack library as well as substitute in sparse retrieval nodes into their existing Haystack systems.
+
+## Installation and Setup ##
+In order to ensure the proper installation, please use only install with python version 3.8.
+
+Install `farm-haystack`'s dependencies via deepsparse extras
+```bash
+pip install deepsparse[haystack]
+```
+
+After this is done, importing assets from `deepsparse.transformers.haystack` will trigger an auto-installation of Neural Magic's fork of `transformers` as well as `farm-haystack[all]==1.4.0`. These auto-installations can be controlled by setting the environment variables `NM_NO_AUTOINSTALL_TRANSFORMERS` and `NM_NO_AUTOINSTALL_HAYSTACK` respectively.
+
```

Comment:
Does this need to be changed based on recent updates to how haystack is installed? We need them to do --no-dependencies right?

Reply:
In the source, haystack is still installed with no-dependencies. Are you suggesting that we note this in the docs?

Reply:
Nevermind, just looked back at the PR I was thinking about. Looks good!